test/api/v01/vrp_test.rb

```diff
@@ -440,7 +444,7 @@ def test_multitrips
 
   def test_count_optimizations
      [
-      { method: 'post', uri: 'submit', operation: :submit_vrp, options: { vrp: VRP.toy }},
```

Comment:
the nickname of the API operation is :submit_vrp

Reply:
I think, even if it is not always natural and we should avoid this, that operation prefix `optimize` is enough explicit to use it. But thread from braktar raises a question for me @alain-andre : why test endpoint without count system ? And in addition you are not testing all endpoints (`: get_job_list` is not here). Maybe you should remove all no counted endpoints in tests...

Reply:
@fab-girard My point of view is that test should cover all possibilities (positives and negatives). I prefer adding `get_job_list` I forgot that remove all other because it makes sure we will never count inside those endpoints. Tell me if I remove uncounted endpoints, or add `get_job_list`.

Reply:
I just noticed I already test `get_job_list`, it's the third in the array.

Reply:
...my bad you were talking in the `when` condition.

Reply:
Ok, I have changed the test, it assert to get count only during optimize, using the `else` condition to validate other endpoints.